pkg/llbutils/utils.go

```diff
@@ -81,7 +81,7 @@ func ImageSource(imageRef string, withMeta bool) llb.State {
 	return llb.Image(imageRef, opts...)
 }
 
-func Copy(src llb.State, srcPath string, dest llb.State, destPath string, chown string) llb.State {
+func Copy(src llb.State, srcPath string, dest llb.State, destPath string, chown string, ignoreCache bool) llb.State {
```

Comment:
The params list starts to be big :thinking: 
But as it's a simple fn, looks there are no choices except refactor to first configure the llbutil to whether use the cache or not :/

So if it's not very an ussue for now, :+1: for additional param

Reply:
Yep, I had the same feeling when adding this parameter :confused: I think this param list will not grow anymore, however if I need to add a new param, I'll switch to a dedicated `struct` :wink: